docs/administration/back_office/back_office_elements/reusable_components.md

```diff
@@ -42,6 +42,85 @@ The alert component has the following properties:
 } only %}
 ```
 
+## Details
+
+The details component consists of the following blocks:
+
+- `details_header`
+
+- `details_items`
+
+Variables:
+
+|Name|Type|Values|
+|----|----|-----------|
+|`headline`|string|
+|`headline_items`|array|
+|`view_mode`|string|`vertical`, default set to `''`|
+|`items`|array|`label`, `content_raw`, `content`|
+
+`headline` and `headline_items` are variables for `@ibexadesign/ui/component/table/table_header.html.twig`
+
+## Modal
+
+The modal component consist of the following blocks:
+
+```html+twig
+{% block dialog %}
+	{% block content_before %}
+	{% block content %}
+		{% block header %}
+		{% block subheader %}
+		{% block body %}
+		{% block footer %}
+	{% block content_after %}
+```
+Variables:
+
+|Name|Type|Values|
+|----|----|-----------|
+|`size`|string|`small`, `large`, `extra-large`, default set to: `''`|
+|`subtitle`|string|default set to `false`|
```

Comment:
It is not by default set to `false`. We have `subtitle|default(false)` in one condition but this is not an assignment of a default value. We can see in the template, that below we have a check whether `subtitle` is defined:
```
{% if subtitle is defined %}
    <div class="ibexa-modal__subheader">
        {{ subtitle }}
    </div>
{% endif %}
```
so I would state that there is no default value and also add that when it is not defined subheader is not rendered.